```
src/main/scala/scalaz/ioeffect/AsyncReturn.scala:34:38: error: [DisableSyntax.covariant] Covariant types could lead to error-prone situations.
```

hahahahahahahahaha.

--- a/build.sbt
+++ b/build.sbt
@@ -12,4 +12,5 @@ addCommandAlias("cpl", "all compile test:compile")
 addCommandAlias("fmt", "all scalafmtSbt scalafmt test:scalafmt")
 addCommandAlias("check",
                 "all scalafmtSbtCheck scalafmtCheck test:scalafmtCheck")
-addCommandAlias("lint", "all compile:scalafixCli test:scalafixCli")
+addCommandAlias("lint", "all compile:scalafixTest test:scalafixTest")
+addCommandAlias("fix", "all compile:scalafixCli test:scalafixCli")

--- a/src/main/scala/scalaz/ioeffect/AsyncReturn.scala
+++ b/src/main/scala/scalaz/ioeffect/AsyncReturn.scala
@@ -31,7 +31,7 @@ object AsyncReturn {
     // avoids object alloc, whilst allowing exhaustivity checking
     // the Later_ subclass is for scala/bug#10822
     // (it makes the pattern matcher think that `Later()` can match `AsyncReturn[A]`)
-    private[this] final class Later_[+A] extends Later[A]
+    private[this] final class Later_[+A /* scalafix:ok */ ] extends Later[A]
     private[this] val value: Later[Nothing] = new Later_[Nothing]
     def apply[A](): AsyncReturn[A]          = value.asInstanceOf[AsyncReturn[A]]
   }

